peerconnection.go

```diff
@@ -1200,7 +1208,7 @@ func (pc *PeerConnection) startReceiver(incoming trackDetails, receiver *RTPRece
 
 	for _, t := range receiver.Tracks() {
 		if t.SSRC() == 0 || t.RID() != "" {
-			return
+			continue
```

Comment:
This is a change from my previous PR to do RTP receiver changes. Realised that `continue` is better code here. Functionally probably makes no difference as all codecs are SSRCs or RIDs.